test/fixtures/application.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const jstp = require('../../');
+
+const name = 'testApp';
+const login = 'login';
+const password = 'password';
+const sessionId = '12892e85-5bd7-4c77-a0c5-a0aecfcbc93a';
+
+const interfaces = {};
+
+const authCallback = (
+  connection, application, strategy, credentials, callback
+) => {
+  if (application.name !== name) {
+    return callback(new jstp.RemoteError(jstp.ERR_APP_NOT_FOUND));
+  }
+
+  let username = null;
+  let success = false;
+
+  if (strategy === 'anonymous') {
+    success = true;
+  }
+
+  if (strategy === 'login' &&
```

Comment:
In my opinion, it could be better to merge this condition with the previous one using `else if`. It can also probably fix the indentation issue mentioned in @aqrln's comment.